intro/intro.rst

```diff
@@ -300,49 +420,46 @@ Other useful magic functions are:
 
   .. sourcecode:: ipython
 
-    In [2]: cd /tmp
+    In [1]: cd /tmp
     /tmp
 
-* ``%timeit`` allows you to time the execution of short snippets using the
-  ``timeit`` module from the standard library:
-
-  .. sourcecode:: ipython
-
-      In [3]: timeit x = 10
-      10000000 loops, best of 3: 39 ns per loop
-
 * ``%cpaste`` allows you to paste code, especially code from websites which has
   been prefixed with the standard Python prompt (e.g. ``>>>``) or with an ipython
   prompt, (e.g. ``in [3]``):
 
   .. sourcecode:: ipython
 
-    In [5]: cpaste
-    Pasting code; enter '--' alone on the line to stop or use Ctrl-D.
-    :In [3]: timeit x = 10
-    :--
-    10000000 loops, best of 3: 85.9 ns per loop
-    In [6]: cpaste
+    In [2]: %cpaste
     Pasting code; enter '--' alone on the line to stop or use Ctrl-D.
-    :>>> timeit x = 10
+    :>>> for i in range(3):
+    :...     print(i)
     :--
-    10000000 loops, best of 3: 86 ns per loop
+    0
+    1
+    2
+
+* ``%timeit`` allows you to time the execution of short snippets using the
+  ``timeit`` module from the standard library:
+
+  .. sourcecode:: ipython
 
+      In [3]: %timeit x = 10
+      10000000 loops, best of 3: 39 ns per loop
 
 * ``%debug`` allows you to enter post-mortem debugging. That is to say, if the
   code you try to execute, raises an exception, using ``%debug`` will enter the
   debugger at the point where the exception was thrown.
 
   .. sourcecode:: ipython
 
-    In [7]: x === 10
+    In [4]: x === 10
       File "<ipython-input-6-12fd421b5f28>", line 1
         x === 10
             ^
     SyntaxError: invalid syntax
 
 
-    In [8]: debug
+    In [5]: %debug
```

Comment:
see also to mention section on debugging?
http://www.scipy-lectures.org/advanced/debugging/index.html